Source/Core/Core/HW/Memmap.cpp

```diff
@@ -379,7 +387,10 @@ void UpdateLogicalMemory(const PowerPC::BatTable& dbat_table)
           void* mapped_pointer = g_arena.CreateView(position, mapped_size, base);
           if (!mapped_pointer)
           {
-            PanicAlertFmt("MemoryMap_Setup: Failed finding a memory base.");
+            PanicAlertFmt(
+                "Memory::UpdateLogicalMemory(): Failed to create view for physical region at "
+                "0x{:08X} (size 0x{:08X}).",
+                intersection_start - mapping_address, mapped_size);
```

Comment:
The physical address here might be wrong, I admit I don't fully understand what this code is doing...